src/cascadia/UnitTests_Control/ControlInteractivityTests.cpp

```diff
@@ -157,7 +158,7 @@ namespace ControlUnitTests
 
             // The mouse location and buttons don't matter here.
             interactivity->MouseWheel(modifiers,
-                                      30,
+                                      -30,

```

Comment:
If it "doesn't matter" then why -30

Reply:
Because the delta actually does matter, and this test never really worked before. This needed to be a `-30` to have the correct delta.